### DISCUSSION TITLE
Why another framework

### DISCUSSION THREAD

**Discussion by underthesand** [dot-agent/nextpy#140] [General]
Naive question: I read your readme and your manifesto and I still don't understand why your project was necessary? What does it offer compared to Reflex?

**Comment by anubrag** [comment]
Our aim is to develop a framework tailored for self-modifying applications. While Reflex is all about web apps, our framework utilizes similar abstractions to facilitate GUI development in Jupyter and enables deployment of core functionalities as bots for platforms like Discord, Slack, and Telegram.

**Reply by BradKML** [reply]
Then what is the difference between this and the other code frameworks then?
- [Anterion](https://github.com/MiscellaneousStuff/anterion) (mixes [SWE-Agent](https://github.com/princeton-nlp/SWE-agent) and [OpenDevin](https://github.com/OpenDevin/OpenDevin))
- [AutoGen](https://github.com/microsoft/autogen)
- [MetaGPT](https://github.com/geekan/MetaGPT)
- [SuperAGI](https://github.com/TransformerOptimus/SuperAGI)
- [CrewAI](https://github.com/joaomdmoura/crewai)
- [GPT-Pilot](https://github.com/Pythagora-io/gpt-pilot)
- [DevOpsGPT](https://github.com/kuafuai/DevOpsGPT)
